lightkurve/tests/test_targetpixelfile.py

```diff
@@ -369,7 +369,6 @@ def test_interact():
     for tpf in [KeplerTargetPixelFile(filename_tpf_one_center),
                 TessTargetPixelFile(filename_tess)]:
         tpf.interact()
-        tpf.interact(lc=tpf.to_lightcurve(aperture_mask='all'))
```

Comment:
Do we expect this to break?  If not, let's keep it in the unit test?

Reply:
Yes, we've deprecated the input custom lightcurve "lc" kwarg with the new pixel selection.